client/web/src/codeintel/ReferencesPanel.tsx

```diff
@@ -601,12 +603,35 @@ const SideBlob: React.FunctionComponent<
             location: H.Location
             history: H.History
```

Comment:
```suggestion
            history: H.History<BlobMemoryHistoryState>
```

And then we shouldn't need the `as any` cast on line 621